src/py_converters.cpp

```diff
@@ -109,8 +109,13 @@ int convert_double(PyObject *obj, void *p)
 int convert_bool(PyObject *obj, void *p)
 {
     bool *val = (bool *)p;
+    int ret;
```

Comment:
just change the signature of convert_bool to `int convert_bool(PyObject *obj, bool *p)`?

Reply:
All of the convert functions use `void*`.

Reply:
not convinced that they should but I guess that's another project...